lib/standard/creates_config_store/assigns_rubocop_yaml.rb

```diff
@@ -18,6 +18,10 @@ def rubocop_yaml_path(desired_version)
         "ruby-1.8.yml"
       elsif desired_version < Gem::Version.new("2.0")
         "ruby-1.9.yml"
+      elsif desired_version < Gem::Version.new("2.1")
```

Comment:
If the minimum parseable version for Rubocop is 2.0, does this mean we need to delete tacit support for 1.8 and 1.9 code from Standard at this point?

Reply:
I would leave the `ruby-1.8.yml` and `ruby-1.9.yml` files.  That way custom Ruby 1.8 and 1.9 rules can be added to those files as needed.  Rubycop might not support linting Ruby 1.8 or 1.9 but Standard can.